spec/values/calculation/round/three_arguments.hrx

```diff
@@ -0,0 +1,241 @@
+a {b: round(up, 10, infinity)}
+
+<===> operation/positive/output.css
+a {
+  b: calc(infinity);
+}
+
+<===>
+================================================================================
+<===> operation/negative/input.scss
+@use "sass:math";
+a {b: round(down, -10, infinity)}
+
+<===> operation/negative/output.css
+a {
+  b: calc(-infinity);
+}
+
+
+<===>
+================================================================================
+<===> operation/two_args/simplify/input.scss
+$c: 10px;
+a {b: round(3.8px + 4px, $c)}
+
+<===> operation/two_args/simplify/output.css
+a {
+  b: 10px;
+}
+
+<===>
+================================================================================
+<===> operation/nearest/simplify/input.scss
+a {b: round(nearest, 115px + 2px, 21px + 4px)}
+
+<===> operation/nearest/simplify/output.css
+a {
+  b: 125px;
+}
+
+<===>
+================================================================================
+<===> operation/up/input.scss
+a {b: round(up, 101px, 25px)}
+
+<===> operation/up/output.css
+a {
+  b: 125px;
+}
+
+<===>
+================================================================================
+<===> operation/down/input.scss
+a {b: round(down, 122px, 25px)}
+
+<===> operation/down/output.css
+a {
+  b: 100px;
+}
+
+<===>
+================================================================================
+<===> operation/to-zero/positive/input.scss
+a {b: round(to-zero, 120px, 25px)}
+
+<===> operation/to-zero/positive/output.css
+a {
+  b: 100px;
+}
+
+<===>
+================================================================================
+<===> operation/to-zero/negative/input.scss
+a {b: round(to-zero, -120px, 25px)}
+
+<===> operation/to-zero/negative/output.css
+a {
+  b: -100px;
+}
+
+<===>
+================================================================================
+<===> infinity/input.scss
+@use "sass:math";
+a {b: round(up, infinity, infinity)}
+
+<===> infinity/output.css
+a {
+  b: calc(NaN);
+}
+
+
+<===>
+================================================================================
+<===> unknown_variable/first_arg/input.scss
+a {
+  e: round(var(--c), 8px, 9px);
+}
+<===> unknown_variable/first_arg/output.css
+a {
+  e: round(var(--c), 8px, 9px);
+}
+
+<===>
+================================================================================
+<===> unknown_variable/third_arg/input.scss
+a {
+  d: round(up, 8px, var(--c));
+}
+<===> unknown_variable/third_arg/output.css
+a {
+  d: round(up, 8px, var(--c));
+}
+
+<===>
+================================================================================
+<===> unknown_number/input.scss
+a {
+  e: round(up, number, 9px);
+}
+
+<===> unknown_number/output.css
+a {
+  e: round(up, number, 9px);
+}
+
+<===>
+================================================================================
+<===> interpolation/input.scss
+a {
+  e: round(#{"up"}, number, 9px);
+}
+
+<===> interpolation/output.css
+a {
+  e: round(up, number, 9px);
+}
```

Comment:
It's less important to test every kind of calculation argument that doesn't simplify to a number (these are all already covered by the `calc()` tests and we can assume that the same simplification logic is in use), and more important to test that _some_ non-number argument works in every argument position in the function.

Reply:
Got it, I replaced `number` with a number with real units. Thank you :)